packages/web-client/app/templates/card-pay.hbs

```diff
@@ -14,7 +14,6 @@
     />
     <CardPay::HeaderNav
       @tabs={{array
-        (hash title="About Card Pay" route="card-pay.index" icon="home")
```

Comment:
We should make sure that visiting `/card-pay` results in a redirect to `card-pay/balances` now.